### ISSUE TITLE
Add a `MSVC2019_COMPATIBILITY` flag to the conan recipe

### ISSUE THREAD

**Issue by mscofield0** [helmesjo/conan-magnum#7]
The recipe fails to install on MSVC 2019 due to this flag being necessary.

**Comment by helmesjo** [comment]
Yeah I remember there being some issues regarding that which weren't trivial to circumvent automagically (requires some changes in the magnum library if I remember correctly (?)).

For now I'd just add the following before finding the package (I use the `cmake_find_package` generator, but the equivalent for any other generator would probably work):
```cmake
set(CORRADE_MSVC2019_COMPATIBILITY ON)
find_package(magnum REQUIRED)
```

**Edit**
It's [`FindCorrade.cmake`](https://github.com/mosra/magnum/blob/047c539a17b9b68d9bf6526b8974bf285956010f/modules/FindCorrade.cmake) which expects that to be set, which is indirectly used by `FindMagnum.cmake` in my case.
